inspire/testsuite/test_export.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+import pkg_resources
+import os
+
+from dojson.contrib.marc21.utils import create_record
+
+from invenio.testsuite import InvenioTestCase, make_test_suite, run_test_suite
+from inspire.dojson.hep import hep
+from invenio.base.wrappers import lazy_import
+Export = lazy_import('inspire.utils.export.Export')
+
+
+class ExportTests(InvenioTestCase):
+
+    def setUp(self):
+        self.marcxml = pkg_resources.resource_string('inspire.testsuite',
+                                                     os.path.join(
+                                                         'fixtures',
+                                                         'test_hep_formats.xml')
+                                                     )
+        record = create_record(self.marcxml)
+
+        self.hep_record = hep.do(record)
+
+        self.sample_export_good = {
+            'citation_key': 'Aad:2015wqa',
+            'doi': '10.1140/epjc/s10052-015-3661-9, 10.1140/epjc/s10052-015-3518-2',
```

Comment:
I am using it like this, the function get_doi does all the job and returns me either one single, either multiple separated by comma.